apps/ai-image-generator/contentful-app-manifest.json

```diff
@@ -6,8 +6,7 @@
       "category": "Custom",
       "description": "This action is used to launch hosted app builds for AIIG.",
       "type": "function",
-      "path": "actions/aiig.js",
-      "entryFile": "./actions/aiig.ts",
+      "path": "backend/actions.js",
```

Comment:
You or Matt will have to explain to me tomorrow why this works without the entry file that points to `actions.ts`

Reply:
But BTW i wouldn't be surprised if I"m wrong here, maybe it wont' work. 🤷 

The main reason I have a hunch I'm right: Our previous entryFile is a typescript file `./actions/aiig.ts` and there are no typescript files in our build. So that's why I believe this is only a compile time thing vs. a runtime thing.

Reply:
As (somewhat) expected `entryFile` is just a compile thing that was needed by the previous `build-actions.js` but not actually required.